src/Symfony/Component/Security/CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ CHANGELOG
 3.4.0
 -----
 
+ * added `TokenInterface` to `\Symfony\Component\Security\Http\Event\SwitchUserEvent` to allow listeners to switch out 
```

Comment:
I think it's more important to document that you can change the token being used by calling the `setToken()` method:

> * added a `setToken()` method to the `SwitchUserEvent` class to allow to replace the created token while switching users